geode-core/src/main/java/org/apache/geode/management/internal/cli/result/CommandResult.java

```diff
@@ -690,19 +691,17 @@ public void setFileToDownload(Path fileToDownload) {
     this.fileToDownload = fileToDownload;
   }
 
-  public List<Object> getColumnValues(String columnName) {
-    Object[] actualValues =
-        toArray(getTableContent().getInternalJsonObject().getJSONArray(columnName));
-    return Arrays.asList(actualValues);
+  public List<String> getColumnValues(String columnName) {
+    return toList(getTableContent().getInternalJsonObject().getJSONArray(columnName));
   }
 
-  private Object[] toArray(JSONArray array) {
+  public static List<String> toList(JSONArray array) {
     Object[] values = new Object[array.length()];
 
     for (int i = 0; i < array.length(); i++) {
       values[i] = array.get(i);
     }
 
-    return values;
+    return Arrays.stream(values).map(Object::toString).collect(Collectors.toList());
```

Comment:
This feels kind of verbose.  I think this would read better.
```
  public static List<String> toList(JSONArray array) {
    String[] values = new String[array.length()];
    for (int i = 0; i < array.length(); i++) {
      values[i] = array.get(i).toString();
    }
    return Arrays.asList(values);
  }
```
Or we could make `JSONArray` iterable, which would give us better functionality of `foreach` loops.